src/MCPClient/lib/clientScripts/upload-archivesspace.py

```diff
@@ -0,0 +1,169 @@
+#!/usr/bin/python
```

Comment:
Probably not! This is the same shebang as upload-atk though... which probably wasn't intentional there